DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -75,31 +79,50 @@ Then run the command `docker images`, you will have the following Docker Images:
 We set default model as "Intel/neural-chat-7b-v3-3", change "LLM_MODEL_ID" in following Environment Variables setting if you want to use other models.
 If use gated models, you also need to provide [huggingface token](https://huggingface.co/docs/hub/security-tokens) to "HUGGINGFACEHUB_API_TOKEN" environment variable.
 
-### Setup Environment Variables
+### Setup Environment Variable
 
-Since the `compose.yaml` will consume some environment variables, you need to setup them in advance as below.
+To set up environment variables for deploying Document Summarization services, follow these steps:
 
-```bash
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
-export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
-export TGI_LLM_ENDPOINT="http://${host_ip}:8008"
-export HUGGINGFACEHUB_API_TOKEN=${your_hf_api_token}
-export MEGA_SERVICE_HOST_IP=${host_ip}
-export LLM_SERVICE_HOST_IP=${host_ip}
-export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/docsum"
-```
+1. Set the required environment variables:
 
-Note: Please replace with `host_ip` with your external IP address, do not use localhost.
+   ```bash
+   # Example: host_ip="192.168.1.1"
+   export host_ip="External_Public_IP"
+   # Example: no_proxy="localhost, 127.0.0.1, 192.168.1.1"
+   export no_proxy="Your_No_Proxy"
+   export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
```

Comment:
```suggestion
   export HUGGINGFACEHUB_API_TOKEN="Your_HuggingFace_API_Token"
```

(This comment applies to all the READMEs in this PR.)

Reply:
Your suggestion has a valid point, but this is how it is used in OPEA (across 21 different README files). To maintain consistency, it should be changed throughout the entire OPEA, which is not within the scope of this PR